casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala

```diff
@@ -158,6 +148,47 @@ object GraphzGenerator {
       node = Map("width" -> "0", "height" -> "0", "margin" -> "0.03", "fontsize" -> "8")
     )
 
+  private def drawParentDependencies[G[_]: Applicative](
+      g: Graphz[G],
+      validators: List[(String, ValidatorsBlocks)]
```

Comment:
Why does this method take `List[(String, ValidatorsBlocks)]` if enough would be `List[ValidatorsBlocks]`?